### PR TITLE
Book editing

### PR DESCRIPTION
**Pull request by k4gi**
#### Summary

```SUMMARY: None```

#### Purpose of change

I've read through all the files in json/items/book, editing as I go. Fixed a few mistakes. Mostly I've expanded the definitions of variants in misc.json onto multiple lines.

#### Describe alternatives you've considered

> "description": "This sheaf of papers--dated two weeks before all this started--describes some new chemical formula, and its effects on human subjects.  It's stamped \"APPROVED\"…",

I put double dashes on this line because a similar book had double dashes. I could have changed the other one to single dashes, this seemed better?

> "description": "This classic reference work contains extensive, dense chapters and tables on materials, metrology, toolmaking, gears, threading and more.  This recent edition includes extensive data on what were the latest techniques in additive manufacturing.  If you need to know how best complete a certain machining operation, the answer lies somewhere in these pages.",

I changed this from "this recent edition includes extensive data on what _was_ the latest techniques", I'm not absolutely sure about this one.

#### Testing

I've copied the edited files over to one of my game installations and checked a few of the books in the debug spawner.

### CODE REVIEW

File: data/json/items/book/maps.json [hunk 1]
```diff
@@ -48,7 +48,17 @@
...
+        "hiway",
+        "road",
+        "bridge",
+        "s_pharm",
+        "s_gun",
+        "s_grocery",
+        "s_hunting",
+        "mil_surplus",
+        "s_gas"
+      ],
```
Comment (Qrox) [inline data/json/items/book/maps.json]: The convention is to put an array or an object in a single line if it's short enough. The JSON formatter will complain if you wrap it this way.

Comment (k4gi) [reply data/json/items/book/maps.json]: i see. i'll change that one back. how short is short enough?

Comment (Qrox) [reply data/json/items/book/maps.json]: I think it's 120 characters, but you can just use the [json linting tool](http://dev.narc.ro/cataclysm/format.html). Copy a changed file into it, hit "lint", and copy the output back.

Comment (snipercup) [reply data/json/items/book/maps.json]: Paste the json in the web linter and it will format it for you. http://dev.narc.ro/cataclysm/format.html

**Comment by k4gi** [comment]
yay it worked this time

**Pull request merged**